### DISCUSSION TITLE
Allow deep search of nodes in CSGCombiner

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4745] [3D]
CSGCombiner doesn't work If I add a Spatial node in it as a pivot.
This could be always allowed or at least an option for the deep search could exist

![изображение](https://user-images.githubusercontent.com/16458555/175772395-33a1d94f-3ac0-4939-8898-57a681a36a6a.png)

That's how it should be
![изображение](https://user-images.githubusercontent.com/16458555/175772550-be91b120-c43e-4ed7-a46e-94c81c333512.png)

**Comment by Calinou** [comment]
You may be able to accomplish this with a RemoteTransform3D node already:

- Keep the CSGSphere3D node in its original position in the scene tree (as a direct child of the CSGCombiner).
- Put a RemoteTransform3D node as a child of the pivot node.
- Set the RemoteTransform3D's target node to the subtractive CSGSphere3D.
